### PR TITLE
Fix pre-publish-check script

### PR DESCRIPTION
**Pull request by kevinbarabash**
## Summary:
In #1099 I moved the source code in each package into 'src', but forgot to update the pre-publish script.  Now the script looks for index.js in the right place.

Issue: none

## Test plan:
- After landing this I'll do a publish.

### CODE REVIEW

File: utils/pre-publish-check.js [hunk 1]
```diff
@@ -3,8 +3,8 @@
  * Pre-publish checks to verify that our publish will go smoothly.
  */
 const path = require("path");
-const glob = require("glob");
 const {exec} = require("child_process");
+const glob = require("glob");
```
Comment (somewhatabstract) [inline utils/pre-publish-check.js:7]: Oooo, we should update this to fast-glob.

No action required.

**Review by somewhatabstract — APPROVED** [review]
LGTM

File: utils/pre-publish-check.js [hunk 2]
```diff
@@ -103,6 +103,12 @@ glob(
                         if (!skipWarnings) {
                             process.exit(1);
                         }
+                        return;
+                    })
+                    .catch((e) => {
+                        console.warn("inquirer prompt failed");
+                        console.warn(e);
+                        process.exit(1);
```
Comment (kevinbarabash) [inline utils/pre-publish-check.js:111]: The linter was complaining about not having `catch` block so I added one.

File: utils/pre-publish-check.js (hunk 1 above)
Comment (kevinbarabash) [inline utils/pre-publish-check.js:7]: The linter auto fixed this.

Comment (kevinbarabash) [reply utils/pre-publish-check.js:7]: I'll update it separately.  It feels like we should almost have a linter for this, lol.

**Pull request merged**